filter_plugins/redis.py

```diff
@@ -0,0 +1,68 @@
+class FilterModule(object):
+    def filters(self):
+        return {
+            'redis_master_address': self.redis_master_address,
+            'redis_master_port': self.redis_master_port,
+        }
+
+    def redis_master_address(self, fact, redis_type='redis', default_address=None, namespace='OPENIO', service=None, monitor_name='OPENIO-master-1'):
+        try:
+            if redis_type == 'redis':
+                return self._master_address_redis(fact, default_address=default_address, namespace=namespace, service=service)
+            elif redis_type == 'redissentinel':
```

Comment:
You're returning inside the `if`s, no need to use `elif` nor `else`. Instead of:
```python
if ...: return ...
elif ...: return ...
else: return ...
```
simply do:
```python
if ...: return ...
if ...: return ...
return ...
```